### DISCUSSION TITLE
Conversion from pystring to char

### DISCUSSION THREAD

**Discussion by subygan** [PyO3/pyo3#3569] [Questions]
When calling rust from python. in a function parameter I have a variable of type `Option<char>`.
This works fine, and fails when more than one character is given in the python string.
But, this pyString to char conversion is not mentioned in the [conversion table](https://pyo3.rs/v0.20.0/conversions/tables)
Does this behaviour come under undefined or, just not documented?

**Comment by davidhewitt** [comment]
I think it's long stable behaviour, just missing from documentation. Would you be willing to open a PR please?

**Comment by subygan** [comment]
That's great! I've created #3570 . If you can think of somewhere else, you want me to add, I can incorporate those changes too.
